src/main/java/echoserver/EchoServerSetup.java

```diff
@@ -0,0 +1,17 @@
+package echoserver;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class EchoServerSetup {
+    public void run(final Integer PORT) throws IOException {
+        ServerSocket hostSocket = new ServerSocket(PORT);
+        HostServer hostServer = new HostServer(hostSocket);
+        Presenter hostScreen = new Presenter(new Console(System.out));
```

Comment:
This `hostScreen` is a type of `Presenter`, but I think a more conventional name for this would be a `Logger`.

Reply:
Further, the `Presenter` has a bit of a wide interface, and I am wondering if some of that behavior could be inlined here. For instance, do we need a method called `showAwaitingInputMessageOnPort`? Or could we just have something like right in this method.

```java
logger.log("Awaiting Input on Port:  %s", port)
```